src/locale/locales/fr/messages.po

```diff
@@ -6941,11 +6942,11 @@ msgstr "Notre politique de confidentialité a été déplacée vers <0/>"
 
 #: src/view/com/composer/state/video.ts:408
 msgid "The selected video is larger than 50MB."
-msgstr "La vidéo sélectionnée a une taille supérieure à 50 Mo."
+msgstr "La vidéo sélectionnée a une taille supérieure à 50 Mo."
```

Comment:
```suggestion
msgstr "La vidéo sélectionnée a une taille supérieure à 50 Mo."
```
And this would restore another non-breaking space that accidentally got removed.

Reply:
i completely missed that one, nice catch